include/bbp/sonata/config.h

```diff
@@ -175,5 +175,106 @@ class SONATA_API CircuitConfig
     std::unordered_map<std::string, PopulationProperties> _edgePopulationProperties;
 };
 
+/**
+ *  Read access to a SONATA simulation config file.
+ */
+class SONATA_API SimulationConfig
+{
+  public:
+    // Parameters defining global simulation settings for spike reports
+    struct Run {
+        // Biological simulation end time in milliseconds
```

Comment:
if you write those comment as docstring compatible they will appear in documentation.

Reply:
docstring compatible is in the format of
```
/**
 * Parameters defining global simulation settings for spike reports
 */
```
?

Reply:
ping @alkino ?

Reply:
yes this is valid syntax, `///` also work for single line

You can have a check after that with `.ci/check_generated_docstring.sh`

Reply:
Thanks! I updated the comments so they are included